mp/src/game/server/momentum/mom_triggers.cpp

```diff
@@ -22,6 +22,28 @@ static MAKE_TOGGLE_CONVAR(mom_triggers_overlay_bbox_enable, "0", FCVAR_DEVELOPME
 static MAKE_TOGGLE_CONVAR(mom_triggers_overlay_text_enable, "0", FCVAR_DEVELOPMENTONLY,
                           "Toggles showing the entity text for momentum triggers, needs map restart if changed!\n");
 
+CON_COMMAND(mom_trigger_teleport_progress, "Teleports player to the last progress trigger\n")
+{
+    CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
+    if (pPlayer)
```

Comment:
Can early return this here and remove the "Player is null" DevWarning, eg.
```
if (!pPlayer)
    return;
```

Reply:
Did this if-statement for better branch prediction but because this code isn't bottle-neck, followed your advice for better readability, thanks a lot!